pkg/common/outcomes/types.go

```diff
@@ -2,15 +2,12 @@ package outcomes
 
 // Outcome is the printable context returned from a check (common.Runnable).
 type Outcome interface {
-	// GetShortStatus returns a short status of the check outcome, such as success, fail, or diagnostic.
-	GetShortStatus() string
+	// GetOutcomeType returns the type/status of the check outcome: pass/fail/info/unknown.
```

Comment:
Nit: // GetOutcomeType returns the type of the check outcome: pass/fail/info/unknown.